### PR TITLE
Adds two more surgerys - Embalming and Body Rejuvenation

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

Adds a spray of Stilizne to each surgery bag
Adds 2 more surgery 

## Why It's Good For The Game

Embalming is a quick way for medical/baddies to hide the body of someone without risk of the smell leaking out and giving the body away, this also helps with rounds that have a lot more corps then normal in morgue

Body Rejuvenation heals some stam, o2 and a lot of toxins, - even for slimes
The goal of this one is for people to be able to better revive using defibs or surgery when cloner is blown or something

## Changelog
:cl:
add: Surgerys and spays in bags
/:cl:

### CODE REVIEW

File: code/modules/surgery/embalming.dm [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	species = list(/mob/living/carbon/human, /mob/living/carbon/monkey)
+	possible_locs = list(BODY_ZONE_CHEST)
+	requires_bodypart_type = 0
+
+/datum/surgery_step/embalming
+	name = "embalming body"
+	implements = list(/obj/item/hemostat = 100, TOOL_SCREWDRIVER = 35, /obj/item/pen = 15)
+	implements = list(/obj/item/reagent_containers/syringe = 100, /obj/item/pen = 30)
+	time = 10
+	chems_needed = list("drying_agent", "sterilizine")
```
Comment (lolman360) [inline code/modules/surgery/embalming.dm:17]: why not formaldehyde?

edit: it's real life embalming fluid iirc

Comment (Trilbyspaceclone) [reply code/modules/surgery/embalming.dm:17]: atm 15u of the fluid does the same, this is a way for quick and easy round start/gear gotten husking

Comment (Trilbyspaceclone) [reply code/modules/surgery/embalming.dm:17]: Also could be a way to no clone trait people :honk:

**Comment by YakumoChen** [comment]
don't body bags automatically stop miasma generation anyway?

**Comment by Trilbyspaceclone** [comment]
// Make corpses rot, emitting miasma
/mob/living/carbon/proc/rot()
	// Properly stored corpses shouldn't create miasma
	if(istype(loc, /obj/structure/closet/crate/coffin)|| istype(loc, /obj/structure/closet/body_bag) || istype(loc, /obj/structure/bodycontainer))
return
Yep

**Review by silicons — APPROVED** [review]

**Pull request merged**